client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -155,8 +155,7 @@ export const ApplicationsTable: React.FC = () => {
 
   const isTaskCancellable = (application: Application) => {
     const task = getTask(application);
-    if (task?.state && task.state.match(/(Created|Running|Ready|Pending)/))
-      return true;
+    if (task?.state && !task.state.match(/(Succeeded|Failed)/)) return true;
```

Comment:
I'm more partial to an array include check in this circumstance, and I even like regex!

For example:
```js
  const isTaskCancellable = (application: Application) => {
    const task = getTask(application);
    return task?.state && !["Succeeded", "Failed"].includes(task.state);
  }
```